cryptography/src/bls12381/primitives/group.rs

```diff
@@ -79,28 +87,65 @@ const BLST_FR_ONE: Scalar = Scalar(blst_fr {
 #[repr(transparent)]
 pub struct G1(blst_p1);
 
+/// The size in bytes of an encoded G1 element.
 pub const G1_ELEMENT_BYTE_LENGTH: usize = 48;
 
+/// Domain separation tag for hashing a proof of possession (compressed G2) to G1.
+pub const G1_PROOF_OF_POSSESSION: DST = b"BLS_POP_BLS12381G1_XMD:SHA-256_SSWU_RO_POP_";
```

Comment:
By default (and in the `bls12381:Scheme` implementation), we use G1 for the Public Key (48 bytes) and G2 for the Signature (96 bytes). `G1_PROOF_OF_POSSESSION` and `G1_MESSAGE` would be used if you flipped those (using G2 for Public Key and G1 for Signature).

I figured it was worth exporting both in case someone preferred to do a threshold construction with smaller signatures (i.e. using G1).